sprint_1_merge v1.0

AronBA opened the pull request:
I am depres...in love with python
:)

AronBA committed:
added multiple cameras

plays sound when sus

Commit:
added sms messaging service
(can be fine tuned)

Klemen-Z committed:
added .gitignore for PyCharm Users

Commit:
Refractor inputs and labels

Klemen-Z committed:
added email sender

not tested yet

Commit:
Change to grid layout

Klemen-Z committed:
added better functionality

Needs mail server to work

Klemen-Z committed:
made email conform to user specs

Klemen-Z committed:
made sms conform to same specs as email

Commit:
Hide and show Email Frame on checkbox click

Commit:
Fix grammar mistake

Commit:
Add Gui for Messenger

It's the same way the email Gui was created

Commit:
Increase size of messenger text

Commit:
Save Mail and Messenger options to settings file

Klemen-Z committed:
Email and SMS now send

keys and passwords not included

Commit:
Add booleans to settings file

Klemen-Z committed:
made sms message shorter

Klemen-Z committed:
Updated email message to have same structure as SMS

AronBA committed:
add support for relais

bug fixxes:
- Motion detection only detects now one time
- playing sound doesn't pause the code
- deactivating the relais dosen't ddos the webserver anymore

AronBA committed:
Merge pull request #3 from AronBA/multiplecam

Multiplecam

AronBA committed:
Merge pull request #4 from AronBA/Messengers

Messengers

AronBA committed:
Merge branch 'updateGui' into development

AronBA committed:
implementation of new features

Klemen-Z reviewed (approved):
My code was merged fine, idk about the rest.

AronBA committed:
added logging
features:
- action will be logged in app.log
- you can enter a debug mode
- you can now quickly change the dev enviroment
- reformatted file
- refactored variables

AronBA committed:
Merge pull request #6 from AronBA/logging

added logging

AronBA committed:
reformat the comments

AronBA committed:
Update README.md
